### DISCUSSION TITLE
PhpSpreadsheet如何读取加密的表格

### DISCUSSION THREAD

**Discussion by Gyh77520082** [PHPOffice/PhpSpreadsheet#2567] [Q&A]
在PHP中如何实现使用PhpSpreadsheet 输入密码，读取已加密的文档

**Comment by oleibman** [comment]
I do not speak, read, or write Chinese, so I will hope that Google Translate is accurate, and that you can also use it with my answer.

PhpSpreadsheet is not able to read or write a password-protected Excel spreadsheet. It does have the ability to password protect individual worksheets.

**Reply by Gyh77520082** [reply]
谢谢你的回答 但是我昨天在观看PhpSpreadsheet文档的时候  看到了Reading protected spreadsheet这一段落 
![1](https://user-images.githubusercontent.com/59432429/153137492-a905cf0f-f5d3-4852-9d47-adf7cbe53dad.png)

 该怎么应用呢
